ml/backend.go

```diff
@@ -26,19 +26,28 @@ type Backend interface {
 	SystemInfo() string
 }
 
-var backends = make(map[string]func(*os.File) (Backend, error))
+// BackendParams controls how the backend loads and executes models
+type BackendParams struct {
+	NumThreads int // Number of threads to use if running on the CPU
 
```

Comment:
```suggestion
```

So the gofmt will kick in, unless you intentionally wanted to separate this

Reply:
I know the comments don't line up but I did actually mean to separate them - the first set is CPU parameters, the second is GPU.